src/lib.rs

```diff
@@ -571,8 +606,29 @@ fn json_obj_len(ctx: &Context, args: Vec<String>) -> RedisResult {
 /// MEMORY <key> [path]
 /// HELP
 ///
-fn json_debug(_ctx: &Context, _args: Vec<String>) -> RedisResult {
-    Err("Command was not implemented".into())
+fn json_debug(ctx: &Context, args: Vec<String>) -> RedisResult {
+    if args.len() < 2 {
+        return Err(RedisError::WrongArity);
+    }
+    let mut args = args.into_iter().skip(1);
+    match args.next_string()?.to_uppercase().as_str() {
+        "MEMORY" => {
+            let key = args.next_string()?;
+            let path = backwards_compat_path(args.next_string()?);
+
+            let key = ctx.open_key(&key);
+            let value = match key.get_value::<RedisJSON>(&REDIS_JSON_TYPE)? {
+                Some(doc) => doc.get_memory(&path)?,
+                None => 0
+            };
+            Ok(value.into())
+        }
+        "HELP" => {
+            let results : Vec<RedisValue> = vec!["MEMORY <key> [path] - reports memory usage".into(), "HELP                - this message".into()];
```

Comment:
It would be nice to make the help text `static` instead of recreating it every time.
Regardless, it would be better to create an array instead of a `Vec` and take a slice of it.
Something like:

```
let results: &[RedisValue] = &["MEMORY <key> [path] - reports memory usage".into(), "HELP                - this message".into()][..];
```

Reply:
Then we might want to change RedisValue::Array(Vec<RedisValue>) to Array([RedisValue])

Reply:
Indeed.